docs/software/media.md

```diff
@@ -96,6 +96,23 @@ Also installs:
 
     Simply select the `Update DB` button from the settings menu in ympd.
 
+=== "Configuration"
+
+    Additional to the ympd configuration via the web interface the underlying MPD can be configured via the file 
+    
+    ```sh
```

Comment:
Practically doesn't make a difference, but since this file path is no shell command:
```suggestion
    ```
```